acapy_plugin_toolbox/holder/v0_1.py

```diff
@@ -107,6 +114,7 @@ async def handle(self, context: RequestContext, responder: BaseResponder):
         session = await context.session()
 
         credentials = await CredExRecord.query(session)
+        credentials = list(filter(self.__filter_credential, credentials))
```

Comment:
I think I'd recommend switching this up a bit:

```python
if self.states:
    credentials = list(filter(lambda cred: cred.state in self.states, credentials))
```

Reply:
Ok, I changed it. I originally split it out to a multi-line function to make room for more filters.